cartridges/int_affirm_controllers/cartridge/templates/default/affirm/paymentmethodinput.isml

```diff
@@ -29,10 +29,9 @@
 			<img alt="${Resource.msg(paymentMethodType.label,'forms',null)}" 
 				src="https://cdn-assets.affirm.com/images/blue_logo-transparent_bg.png" 
 				title="${Resource.msg(paymentMethodType.label,'forms',null)}"/>
-			<span>${Resource.msg('payment.monthlypayments', 'affirm', null)}</span>
-			<a class="affirm-product-modal" style=""
-			data-amount="${pdict.Basket.totalGrossPrice.multiply(100).getValue().toFixed()}"
-			>${Resource.msg('payment.learnmore', 'affirm', null)}</a>
+			<span>${Resource.msg('payment.payovertime', 'affirm', null)}</span>
+			<a data-page-type="checkout" data-amount="${pdict.Basket.totalGrossPrice.multiply(100).getValue().toFixed()}"
```

Comment:
Do we want to keep the class and just add the data-page-type

Reply:
Yes, it makes sense to keep the class. I've restored it for all three versions (SFRA, Pipelines, Controllers)